pkg/compiler/testdata/errors.txt

```diff
@@ -69,7 +69,7 @@ resource r6[int64be]		### int64be can't be resource base (int types can)
 resource r9["foo"]		### unexpected string "foo", expect type
 
 foo$7(a r0, a1 r2[opt])
-foo$8(a fileoff[a, b, c])	### wrong number of arguments for type fileoff, expect no arguments
+foo$8(a fileoff[a, b, c])	### template fileoff needs 1 arguments instead of 3
```

Comment:
I assume that if I do 1 argument, then it will say "needs 0 arguments instead of 3" in this context. How hard it is to say 0 here?

Reply:
Not sure what you mean by doing 1 argument, but I've changed the error message to say "template fileoff needs 1 or 0 arguments instead of 3".

Reply:
I mean:
if I see this error message and change the description to:
```
foo$8(a fileoff[int32])
```
then I still should get "needs 0 arguments instead of 1" error message.